apps/server/src/models/transactions.ts

```diff
@@ -69,25 +71,24 @@ Transactions.init(
       allowNull: false,
     },
     status: {
-      type: DataTypes.ENUM(...Object.values(transaction_status)),
+      type: DataTypes.ENUM(...Object.values(transactionStatus)),
       allowNull: false,
     },
     action: {
       type: DataTypes.ENUM(...Object.values(action)),
       allowNull: false,
     },
     type: {
-      type: DataTypes.ENUM(...Object.values(transaction_type)),
+      type: DataTypes.ENUM(...Object.values(transactionType)),
       allowNull: false,
     },
-    receiver: {
+    receiverId: {
       type: DataTypes.STRING,
-      allowNull: false,
+      allowNull: true,
     },
-    created_at: {
-      type: DataTypes.DATE,
-      allowNull: false,
-      defaultValue: DataTypes.NOW,
```

Comment:
As you removed the createdAT, shouldn't you add timestamps to the table, or the timestamps are not needed? I think we should know when an action take place in the db for clear record sake. What do you think?